cli/006-cli-output-options.md

```diff
@@ -0,0 +1,36 @@
+- Start Date: 2021-06-11
+- RFC PR:
+- Issue:
+
+### Improve commands output UX and rendering options
+
+Current Issues and improvement proposals:
+
+#### - Many commands do not support switching between Table and JSON views.
+
+In places where we retrieve and print entity objects, support changing how the objects are printed through `--json` and `--table` flags. For arrays of objects, usually prefer Table view as the default. Implement the proposal by creating a unified rendering utility.
+``` bash
+tctl workflow list --table # default for most commands
+tctl workflow list --json
```

Comment:
Should it be more like --output-format with _table_ being the default?

Reply:
right, there was a discussion around this in other comments and already implemented in this way using `--format` flag
https://github.com/temporalio/tctl/pull/25

updating this proposal